### PR TITLE
22century page html complete

### PR DESCRIPTION
**Pull request by Mzojobe**
22 century page HTML complete

### CODE REVIEW

**Review by qod-dante — APPROVED** [review]
I approve of everything, with the exception of the "Payfast Logo White.png". The file name must be changed to have the spaces be replaced by dashes or underscores. Same must be done in the code.

**Review by codedilla — CHANGES_REQUESTED** [review]
1. re-naming of the PayFast logo.
2. white spaces must be closed line 71, 80, 83, 85 and 86

**Review by qod-dante — APPROVED** [review]
With the conflicts corrected, I give this 1000% approval.

**Review by codedilla — APPROVED** [review]

File: pages/22century.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>22centuryeducation</title>
```
Comment (qodvilla) [inline pages/22century.html]: Add space to the title.
like so, 22 Cent...

**Review by qodvilla — CHANGES_REQUESTED** [review]

File: pages/22century.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+<body>
+    <div><img src="/images/IA_Logo_2.png" height="50px"></div>
+    <main>
+        <div>
+            <h1>22 century education</h1>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Commodi corrupti enim quisquam vitae dolorum, ut, neque distinctio, animi debitis quibusdam repudiandae harum ab nobis sequi iste? Nisi amet repellat quaerat?</p>
+        </div>
+        <div>
+           <i class="ri-plug-fill ri-2x"></i>
+           <h4>work seminars</h4>
```
Comment (qodvilla) [inline pages/22century.html]: Workshop Seminars

**Review by qodvilla — CHANGES_REQUESTED** [review]

**Review by qod-dante — APPROVED** [review]
All the mistakes that @qodvilla pointed have been fixed this time

File: pages/22century.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+<body>
+    <div><img src="/images/IA_Logo_2.png" height="50px"></div>
+    <main>
+        <div>
+            <h1>22 CENTURY EDUCATION</h1>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Commodi corrupti enim quisquam vitae dolorum, ut, neque distinctio, animi debitis quibusdam repudiandae harum ab nobis sequi iste? Nisi amet repellat quaerat?</p>
+        </div>
+        <div>
+           <i class="ri-plug-fill ri-2x"></i>
+           <h4>Workshop seminars</h4>
```
Comment (qodvilla) [inline pages/22century.html]: Please also update this be all caps.

**Review by qodvilla — CHANGES_REQUESTED** [review]

**Review by qod-dante — APPROVED** [review]
It's done.

**Review by qod-dante — COMMENTED** [review]
He's correcting it one last time.

**Review by qod-dante — APPROVED** [review]

**Review by codedilla — APPROVED** [review]

**Review by qodvilla — APPROVED** [review]

**Pull request merged**